tutorials/applications/napari_imageJ.md

```diff
@@ -1,148 +0,0 @@
-# napari + ImageJ how-to guide
```

Comment:
This document now lives in the how-to guides, so it was moved in the PR to napari/napari.